addons/mail/static/src/messaging/component/chatter/chatter.scss

```diff
@@ -7,13 +7,34 @@
     flex: 1 1 auto;
     flex-direction: column;
     overflow: auto;
-    padding: 0;
 }
 
 .o_FormRenderer_chatterContainer {
     display: flex;
     flex: 1 1 auto;
-    width: 530px;
+    margin-left: auto;
+    margin-right: auto;
+    width: 100%;
+
+    @media (min-width: 576px) {
+        max-width: 540px;
+    }
+
+    @media (min-width: 768px) {
+        max-width: 720px;
+    }
+
+    @media (min-width: 992px) {
+        max-width: 960px;
+    }
+
+    @media (min-width: 1200px) {
+        max-width: 1140px;
+    }
+
+    @media (min-width: 1534px) {
+        width: 530px;
+    }
```

Comment:
not that great, but that's the best you can do without changing style rules on form view, so let's keep it like this

Reply:
:+1: